pkgs/development/libraries/gaia/default.nix

```diff
@@ -0,0 +1,81 @@
+{ stdenv
+, lib
+, fetchFromGitHub
+, libyaml
+, swig
+, pkgconfig
+, wafHook
+, makeWrapper
+, qt4
+, python
+, pythonSupport ? true
+# Default to false since it brakes the build
```

Comment:
```suggestion
# Default to false since it breaks the build
```

What's the reason for the build failure?

Reply:
See https://github.com/MTG/gaia/issues/11, I've added this link in the comment.